tx-source/dash_core.php

```diff
@@ -752,6 +752,7 @@
 $Definition['Save Colors'] = 'Save Colors';
 $Definition['Score Count'] = 'Score Count';
 $Definition['Search by user or role.'] = 'Search for users by name or email, optionally using % as a wildcard. You can also search by user ID, the name of a role, or &ldquo;banned&rdquo;.';
+$Definition['Search Result in New Tab'] = 'Search Result in New Tab';
```

Comment:
```suggestion
$Definition['Search Result In New Tab'] = 'Search Result In New Tab';
```
The casing should be uniquely followed